src/test/java/org/sagebionetworks/web/unitclient/widget/profile/UserProfileEditorWidgetImplTest.java

```diff
@@ -239,9 +239,23 @@ public void testConfigureNoEmail() {
   @Test
   public void testConfigureNotOwner() {
     when(mockAuthController.getCurrentUserPrincipalId()).thenReturn(null); //anonymous
+    profile.setEmails(null);
+    widget.configure(profile, ORC_ID, mockCallback);
+
+    verifyConfigure(1);
+    verify(mockView, never()).setEmails(anyList(), anyString());
+    verify(mockView).setCanEdit(false);
+  }
+
+  @Test
+  public void testConfigureACT() {
```

Comment:
If ACT, then the emails should now be returned, but we are unable to get the notification email.